### PR TITLE
Ignore Freshmaker events triggered by manual rebuilds in dry run mode

### PR DESCRIPTION
**Pull request by sarah256**
From [FACTORY-4870](https://projects.engineering.redhat.com/browse/FACTORY-4870)

### CODE REVIEW

**Comment by hanzz** [comment]
You should check the `dry_run` field, not the `manual_rebuild`.

**Comment by sarah256** [comment]
> You should check the `dry_run` field, not the `manual_rebuild`.

@hanzz Do you know when this field was added? I see that Datagrepper has this now but I don't believe that it used to, since our test message does not include it.

**Comment by hanzz** [comment]
It has been added 9 months ago here: https://pagure.io/freshmaker/c/6b627c03e6619281498166d549036dd237e23b50

**Comment by sarah256** [comment]
@mprahl Do you think it would be better to just add a dry_run attribute to the models, or do it like this (and in the scrapers probably too)?

File: estuary_updater/handlers/freshmaker.py [hunk 1]
```diff
@@ -51,6 +51,10 @@ def event_state_handler(self, msg):
         :param dict msg: a message to be processed
         """
         msg_id = msg['body']['msg']['message_id']
+
+        if 'dry_run' in msg['body']['msg'] and msg['body']['msg']['dry_run']:
```
Comment (mprahl) [inline estuary_updater/handlers/freshmaker.py]: Could you do `if msg['body']['msg'].get('dry_run'):`?

[the remark this answers is not in the document]
Comment (yashvardhannanavati) [reply estuary_updater/handlers/freshmaker.py]: Ah, I see Matt already suggested it. Resolving this.

**Pull request merged**